tests/codegen/consts.rs

```diff
@@ -9,7 +9,7 @@
 // CHECK: @STATIC = {{.*}}, align 4
 
 // This checks the constants from inline_enum_const
-// CHECK: @alloc_76bfe2f13a3e3b01074971d122eac57e = {{.*}}, align 2
+// CHECK: @alloc_701ed935fbda2002838d0a2cbbc171e5 = {{.*}}, align 2
```

Comment:
In this test and the one below, I've had to change the expected name for some allocations. 

My intuition is that the layout of masks changed and that changed the hash (if that's one) used in these symbols, is that correct @oli-obk ? I'm not familiar with how the constants get their names. 

I'm guessing that checking the full constants' symbols is not common in codegen tests, and that's why only these two needed changing.

Reply:
yes, that is entirely expected due to the reasons you intuited.